test/callbacks.js

```diff
@@ -84,6 +96,24 @@ test('validCallback', function(t) {
     field.setValid(true);
 });
 
+test('on valid', function(t) {
+    var field = new FakeField({valid: false});
+    var count = 2;
+    var form = new FormView({
+        fields: [ field ]
+    });
+
+    form.on('valid', function(validBool) {
+	     if (--count <= 0) {
```

Comment:
indentation.  decrement in comparison works, but you could debatably more cleanly set a `t.plan(2)` at the top of your test, then put `t.end()` at the end, after `field.setValid(true);`.  Then no worrying about counting state